docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
```

Comment:
It is nothing critical but just have a look at line 3 and 6. The differences are across the whole document.